utils/RocketCaller.ts

```diff
@@ -0,0 +1,79 @@
+import { IHttp, IRead } from '@rocket.chat/apps-engine/definition/accessors';
+
+export class RocketCaller {
+    public static xAuthToken = '';
+    public static xUserId = '';
+
+    public static async rocketDepartmentIdFromName(read: IRead, http: IHttp, departmentName: string): Promise<string | null> {
```

Comment:
I suggest the following name changes here:

`rocketDepartmentIdFromName` -> `departmentIdFromName` (because the scope is RocketCaller, so depatment functions only are from Rocket)
`departmentName` -> `name` (because the same, `name` only can be the department name)